docs/fast_data/single_view_trigger_generator.md

```diff
@@ -8,29 +8,59 @@ sidebar_label: Single View Trigger Generator
+- operations occurring during initial load phase are easier to handle and carry out
+- processing load can be better spread across the two services depending on the requirements, so that it is possible to optimize the different number of services' replicas depending on the ongoing situation
+- the projection changes identifier are handled only within the event streaming platform (via `sv-trigger` events), so that they don't require additional space on the database
 
-Last thing worth mentioning is the Single View Trigger Generator couples up with the Single View and not with the System of Records, therefore for each Single View Creator you have in your project you will need a Single View Trigger Generator.
+### Migration Guide for adopting Single View Trigger Generator
 
-### What do I need to change in my current setup?
+In order to introduce the Single View Trigger Generator in the Fast Data flow it is necessary to update the configuration of the Real-Time Updater and the Single View Creator. Below are reported the steps for each service.
 
-In order to make the Single View Trigger Generator work you will first need to [enable the emission of Projection Update messages](/fast_data/configuration/realtime_updater/common.md#kafka-projection-updates-configuration) from the Real Time Updater, as it is disabled by default. After that you might as well disable the Projection Changes generation since it will now be the Single View Trigger Generator's job to do it. To do so you simply need to set the `PROJECTIONS_CHANGES_ENABLED` environment variable to `false` in your Real Time Updater.
+#### Real-Time Updater
 
-Last step is to make the Single View Creator read the Projection Changes from a Kafka topic. In order to do that you need to set/update the `PROJECTIONS_CHANGES_SOURCE` environment variable to `KAFKA` and the `KAFKA_PROJECTION_CHANGES_TOPICS` environment variable to the topic where the `sv-trigger` message will be sent. If you don't know what topic name you should use, check out the [naming convention](/fast_data/inputs_and_outputs.md#topic-naming-convention-2).
+- set environment variable `GENERATE_KAFKA_PROJECTION_UPDATES` to `true` to enable the production of `pr-update` events
+- set environment variable `KAFKA_PROJECTION_UPDATES_FOLDER` to `/home/node/app/kafkaProjectionUpdates`. This allows the Real-Time Updater to load a configuration file automatically generated that instruct the service how to map each projection to the corresponding projection-update topic of the event streaming platform. This mapping can be changed in Console in each projection detail section, besides the configuration of the ingestion topic. In case this config map is not already available, please commit your work. It should appear afterward.
```

Comment:
```suggestion
- set environment variable `KAFKA_PROJECTION_UPDATES_FOLDER` to `/home/node/app/kafkaProjectionUpdates`. This allows the Real-Time Updater to load a configuration file automatically generated that instructs the service how to map each projection to the corresponding projection-update topic of the event streaming platform. This mapping can be changed in Console in each projection detail section, besides the configuration of the ingestion topic. In case this config map is not already available, please commit your work. It should appear afterward.
```